### PR TITLE
opencensus: tweak enum in configuration API.

### PR DESCRIPTION
**Pull request by htuch**
Since this API is still experimental, tweaking to match best proto
practices.

Risk level: Low

Signed-off-by: Harvey Tuch <htuch@google.com>

### CODE REVIEW

**Comment by htuch** [comment]
@g-easy @curiouserrandy as discussed on e-mail thread. I can add tests if you think it's worthwhile for the null op as well if this is the right behavior.

File: test/extensions/tracers/opencensus/tracer_test.cc [hunk 1]
```diff
@@ -166,12 +166,12 @@ TEST(OpenCensusTracerTest, PropagateTraceContext) {
   // The test calls the helper with each kind of incoming context in turn.
   auto helper = [](const std::string& header, const std::string& value) {
     OpenCensusConfig oc_config;
-    oc_config.add_incoming_trace_context(OpenCensusConfig::trace_context);
-    oc_config.add_incoming_trace_context(OpenCensusConfig::grpc_trace_bin);
-    oc_config.add_incoming_trace_context(OpenCensusConfig::cloud_trace_context);
-    oc_config.add_outgoing_trace_context(OpenCensusConfig::trace_context);
-    oc_config.add_outgoing_trace_context(OpenCensusConfig::grpc_trace_bin);
-    oc_config.add_outgoing_trace_context(OpenCensusConfig::cloud_trace_context);
+    oc_config.add_incoming_trace_context(OpenCensusConfig::TRACE_CONTEXT);
```
Comment (g-easy) [inline test/extensions/tracers/opencensus/tracer_test.cc]: Optional: add NONE as both incoming and outgoing to smoke-test that it's ignored.

**Comment by g-easy** [comment]
This change looks good to me. Up to you if you want to add NONE to the test. :) Thanks!

File: test/extensions/tracers/opencensus/tracer_test.cc (hunk 1 above)
Comment (htuch) [reply test/extensions/tracers/opencensus/tracer_test.cc]: Done

**Pull request merged**